.github/workflows/check-artifacthub-tags.yml

```diff
@@ -0,0 +1,94 @@
+# Check if all gadgets have tagged official and cncf in artifacthub.
+name: Check ArtifactHub Tags of Gadgets
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '*'
+      - 'gadgets/**'
+  pull_request:
```

Comment:
You're right, new gadgets might not be on ArtifactHub yet. 
Should we only trigger this when there's a push to the `main` branch for the `gadgets/**` path?

Also, I’ll remove the `*` from paths - I added them just for testing.

Regarding the ratelimits:
I'm not sure of the exact numbers, as they weren’t mentioned anywhere in the docs  ([inspektor-gadget api docs](https://artifacthub.io/docs/api/#/Packages/getInspektorGadgetDetails)).
Thanks for the review :)

Reply:
> Should we only trigger this when there's a push to the main branch for the gadgets/** path?

Good question, I feel we should skip running the check on push/pull_request because the CI will always be red on merging a new gadget which is annoying. Running it daily is a [good compromise](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/.github/workflows/scorecard-analysis.yml#L9). What do you say?

Reply:
Yeah ok, that makes sense. I added schedule trigger to workflow, after removing push/pull_request triggers.
Also added delay between artifacthub requests, to gracefully handle rate limits (as we discussed).

Thanks for the suggestion.